src/Native/MatrixFactorizationNative/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+project (MatrixFactorizationNative)
+
+include_directories(libmf)
+
+set(SOURCES
+    UnmanagedMemory.cpp
+    libmf/mf.cpp
+)
+
+if(NOT WIN32)
+    list(APPEND SOURCES ${VERSION_FILE_PATH})
+endif()
+
+if(APPLE)
+    include_directories(/usr/include/malloc) # for malloc.h
```

Comment:
Why is this necessary? I haven't see anyone else including `/user/include/malloc` in their CMake files.

Reply:
Instead, I HAVE seen this pattern in C code:

```
#ifdef __APPLE__

#include <malloc/malloc.h>

#endif // __APPLE__
```

---
In reply to: [227591868](https://github.com/dotnet/machinelearning/pull/1263#discussion_r227591868) [](ancestors = 227591868)

Reply:
This SO says that malloc.h is deprecated. [https://stackoverflow.com/questions/12973311/difference-between-stdlib-h-and-malloc-h](https://stackoverflow.com/questions/12973311/difference-between-stdlib-h-and-malloc-h)

---
In reply to: [227592034](https://github.com/dotnet/machinelearning/pull/1263#discussion_r227592034) [](ancestors = 227592034,227591868)

Reply:
Just successfully removed it.